src/dotnet/ToolManifest/LocalizableStrings.resx

```diff
@@ -0,0 +1,147 @@
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="assembly">
+            <xsd:complexType>
+              <xsd:attribute name="alias" type="xsd:string" />
+              <xsd:attribute name="name" type="xsd:string" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="data">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="CannotFindAnyManifestsFileSearched" xml:space="preserve">
+    <value>Cannot find any manifests file. Searched {0}</value>
+  </data>
+  <data name="FieldCommandsIsMissing" xml:space="preserve">
+    <value>field commands is missing</value>
+  </data>
+  <data name="InvalidManifestFilePrefix" xml:space="preserve">
+    <value>Invalid manifest file. {0}</value>
+  </data>
+  <data name="IsRootFalseNotSupported" xml:space="preserve">
+    <value>isRoot is false is not supported.</value>
+  </data>
+  <data name="MissingVersion" xml:space="preserve">
+    <value>field version is missing</value>
+  </data>
+  <data name="PackageNameAndErrors" xml:space="preserve">
+    <value>In package {0}: {1}.</value>
+  </data>
+  <data name="TargetFrameworkIsUnsupported" xml:space="preserve">
+    <value>TargetFramework {0} is unsupported</value>
+  </data>
+  <data name="Version1NotSupported" xml:space="preserve">
+    <value>version that is not 1 is not supported.</value>
```

Comment:
fixed, changed suggested, and it is a temp message, so I made it inline string

Reply:
This one I don' think should be a temp message, I think it should be an error to try to restore a newer format than the SDK understands

Reply:
Agree, we should surface warning(instead of error). However, that message is a blind block. The logic should be bigger than 1 and the warning should be similar to "if you experience problem, please update your dotnet SDK version"